breadcord/module.py

```diff
@@ -61,6 +61,14 @@ async def unload(self) -> None:
         self.loaded = False
         self.logger.info('Module successfully unloaded')
 
+    async def reload(self) -> None:
+        self.loaded = False
+        self.load_settings_schema()
+        self.install_requirements()
+        await self.bot.reload_extension(self.import_string)
+        self.loaded = True
```

Comment:
I don't know if this ephemeral toggling or `self.loaded` to False is necessary, but async code can get real fiddly so it's probably best to leave it in just in case unless we know for certain it won't goof anything up.